### ISSUE TITLE
Add icons

### ISSUE THREAD

**Issue by kristinashu** [mozilla/mofo-bootstrap#91]
@flukeout is gathering all of our icons but I'm wondering how we should implement them on to our sites?

Should we make them into an icon font or use them as individual SVGs? Would we add them to bootstrap?

cc @gvn @alanmoo @sabrinang @taisdesouzalessa

**Comment by gvn** [comment]
My instinct is to not make icons part of `mofo-bootstrap`.

I think having a separate Icon font would probably make sense.

**Comment by flukeout** [comment]
Heck yes @gvn - icon fonts all day every day. We COULD continue work on https://github.com/flukeout/mozmaker-icon-font

**Comment by gvn** [comment]
That seems like a good idea! Can we rename it to `mofo-icon-font` or something `mofo-` prefixed?

**Comment by flukeout** [comment]
Oh for sure, name is up for grabs and we can move it to a mozilla repo. "mozmaker" haha.

**Comment by taisdesouzalessa** [comment]
We could add those icon fonts to our pattern library so we have a central place to grab them for design comps. Here is an example
![icon-fonts](https://cloud.githubusercontent.com/assets/22157921/20362162/62e5a73c-abef-11e6-9931-c428665aab5a.jpg)

@flukeout please let me know where I can grab the font file for those icons.

**Comment by kristinashu** [comment]
@taisdesouzalessa I think when we're using icons in comps we would use the icon font. This way we only need to maintain the one file (rather than the font and the Craft Library). Using the font in comps might also help dev determine the individual icon font sizes.

**Comment by flukeout** [comment]
Cool, I combed through all of our sites and added all of the icons into the icon font I created a while back. Here they all are...

* https://flukeout.github.io/mozmaker-icon-font/

You can right-click to download each ``.svg`` file as well.

Let me know if something is broken or missing.

**Comment by flukeout** [comment]
There are still a bunch of icons in Thimble that I haven't incorporated. So I'll work on that next, they're listed in...

https://github.com/flukeout/mozmaker-icon-font/issues/4

**Comment by taisdesouzalessa** [comment]
@kristinashu we would incorporate the icons in the font format in the Craft Library (I think my screenshot was not clear about it, sorry for that). I agree with you it should be a font so dev knows the size and Craft allows that. The second point you raised, about having to keep 2 files...on one hand I agree it is work to keep 2 files but on the other hand I think having all the assets in one place (like the Library) speeds up the process while we are working in the comps (instead of opening a site, copy the font and paste it in the file) - I think the benefit is greater than the cost. We could analyze the pros and cons in our weekly meeting? Or try one way or the other and tweak as we go.

**Comment by flukeout** [comment]
Might move this stuff into https://icomoon.io instead of Glyphter (to generate the font file), we have too many icons for the free version of Glyphter. This might change how the icons are used, will investigate.

**Comment by alanmoo** [comment]
https://cloudfour.com/thinks/seriously-dont-use-icon-fonts/



*runs away*

**Comment by flukeout** [comment]
Interesting read @alanmoo - I'm happy to experiment with SVGs, one of the best things about an icon font is the ability to change the color of the icon easily, if we can achieve that with SVGs - then that would be a big win. Have you tried it? I haven't experimented.

Another benefit is that they auto-size to the content you're embedding them into, which is nice for icons beside text. Can we address that too?

I'm already collecting all of the SVGs we're using as part of this effort, so we could expose a folder full of svgs and give people easy to copy markup snippets to include them in their pages.

**Comment by flukeout** [comment]
Looks like some of that stuff is addressed in the article - I'll dig around some more.

**Comment by alanmoo** [comment]
Yeah, I know there are useful cases for icon fonts like that, my biggest concern is the accessibility issues. (Unless you can pull off whatever magic https://symbolset.com/ is doing).

That being said, I unfortunately don't have the time to dig into this, I just wanted to bring up the topic in case someone else happens to have the bandwidth to look into it.

**Comment by taisdesouzalessa** [comment]
Thanks for sharing this article, @alanmoo. It made me think about so many other issues I didn't consider. I agree that accessibility is a big issue in favour of not using the icon font.
@flukeout about changing colours of svgs, maybe this article can help? https://css-tricks.com/cascading-svg-fill-color/

**Comment by alanmoo** [comment]
Keep in mind there's probably ways to automate these things/make them easier to use in the implementation side, it's just a matter of sorting that out and building it into our repos. For the design side, it may still be easier to use fonts, I just want to make sure whatever we do meets the needs of not just us but our users.

**Comment by kristinashu** [comment]
@flukeout created a repo with the icon font https://github.com/mozilla/foundation-icons
